aiographite/aiographitesend.py

```diff
@@ -0,0 +1,449 @@
+import asyncio
+from aiographite.graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
```

Comment:
I'd probably name this "AIOGraphiteSendService" or something, instead of AsyncIOGraphite. I know it doesn't match up with the asyncio name, but everything else in this module (egg name, exception name, etc) uses "AIO"